.github/workflows/ci.yml

```diff
@@ -13,12 +13,15 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        ruby-version: ['3.0.4']
+        ruby-version: ['3.0.4', '3.1.0']
         mongodb-version: ['4.2.14']
```

Comment:
Do we still need to support both?

I think we need to remove `3.0.4` and regenerate `Gemfile.lock` (for Ruby 2) note currenlty we have two `Gemfile.lock` and `Gemfile3.lock` (For ruby 3..0.4) I think we longer need to have both, unless this still support Ruby 2 and we need to regeerate one of them with the new version

Reply:
Hi @ghassanmas
We do not need to support both. We will have a cleanup of the `Gemfile` after the `3.1.0` upgrade work is complete and tested. 
Then we will have a single `Gemfile` for ruby `3.1.0`, and single CI for `3.1.0`. There will be a followup PR for that!